src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -156,6 +156,7 @@ RUN /usr/bin/echo -e '\n--enable-fips' >> config/node.options
 RUN /usr/bin/echo '--openssl-config=/usr/share/kibana/config/nodejs.cnf' >> config/node.options
 COPY --chown=1000:0 openssl/nodejs.cnf "/usr/share/kibana/config/nodejs.cnf"
 ENV OPENSSL_MODULES=/usr/share/kibana/openssl/lib/ossl-modules
+ENV XPACK_SECURITY_FIPSMODE_ENABLED=true
```

Comment:
For some reason, this env variable works fine in the Docker image, but isn't picked up in CI.

Reply:
@kc13greiner Do you have any idea why this wouldn't work in CI?

Reply:
I think it needs `EXPERIMENTAL` from the `xpack.security.experimental.fipsMode.enabled` in `src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker`